### ISSUE TITLE
[Feature Request] Set default theme 

### ISSUE THREAD

**Issue by tekwarfare** [alexta69/metube#171]
Is there/Can there be a way of setting the default theme to always be dark mode?

**Comment by alexta69** [comment]
Once you switch the mode, the setting remains saved in the browser in a cookie, so when you use the same browser, it should use the same mode.

**Comment by tekwarfare** [comment]
Thanks for getting back to me. Having this as a setting would enforce dark-mode for new users without having to tell them to switch.

**Comment by alexta69** [comment]
I guess an environment variable can be added to set the default. Leaving the issue open in case someone wants to submit a PR.

**Comment by PikuZheng** [comment]
try to enter into the container (use nsenter) and execute this command:
```
sed -i '$a html{filter:invert(1) hue-rotate(180deg)}' /app/ui/dist/metube/styles.*.css
```
I tried to do it by using ENV and docker-entrypoint.sh but no permissions. fyi

**Comment by vkartk** [comment]
Hey @alexta69 Looks like #171 , already resolved issue in  #279 !

**Comment by alexta69** [comment]
fixed by #279
